src/rosdiscover/recover/symbolic.py

```diff
@@ -122,21 +124,26 @@ def __str__(self) -> str:
         return self.value
 
     @classmethod
-    def from_name(cls, name: str) -> SymbolicValueType:
-        name_to_type = {
+    def name_to_type(cls):
+        return {
             "bool": cls.BOOL,
             "integer": cls.INTEGER,
             "node-handle": cls.NODE_HANDLE,
             "string": cls.STRING,
             "float": cls.FLOAT,
             "unsupported": cls.UNSUPPORTED,
         }
-        if name not in name_to_type:
+
+    @classmethod
+    def from_name(cls, name: str) -> SymbolicValueType:
+
+        if name not in cls.name_to_type():
             raise ValueError(f"unknown value type: {name}")
-        return name_to_type[name]
+        return cls.name_to_type()[name]
 
 
-class SymbolicValue(abc.ABC):
+class SymbolicExpr(abc.ABC):
+
```

Comment:
again: remove excess blank line, especially since it dislocates the class header from the docstring

Reply:
fixed

Reply:
fixed